Dockerfile

```diff
@@ -2,21 +2,18 @@ FROM java:openjdk-8
 ENV MAXWELL_VERSION 1.10.7
 ENV KAFKA_VERSION 0.10.1.0
 
-RUN apt-get update && apt-get -y upgrade
-
-RUN apt-get install build-essential ruby -y
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get install build-essential maven ruby -y
```

Comment:
Is `ruby` necessary anymore with the introduction of `maven`? I've removed it from my forked maxwell Dockerfile entirely.

Reply:
@nebez Good call. I think ruby is only needed for the release script. Running a build now without it 🤞